packages/blade/src/components/Accordion/Accordion.tsx

```diff
@@ -37,12 +41,26 @@ type AccordionProps = {
 } & TestID &
   StyledPropsBlade;
 
+const MIN_WIDTH: BoxProps['minWidth'] = {
+  s: makeSize(size[200]),
+  m: makeSize(size[360]),
+  l: makeSize(size[400]),
+};
+
+const MAX_WIDTH: BoxProps['maxWidth'] = {
+  s: `calc(100vw - ${makeSize(size[40])})`,
```

Comment:
Can you add a comment on what's this calc doing?

Reply:
Will add it does a `viewport width - 40px`

Reply:
Done in #1296